tests/main/try-snap-goes-away/task.yaml

```
@@ -34,11 +34,9 @@ execute: |
     echo And is gone afterwards
     not snap list "$SNAP_NAME"
 
-    #shellcheck source=tests/lib/dirs.sh
-    . "$TESTSLIB"/dirs.sh
-
     echo And all its binaries
-    N="$(find $SNAP_MOUNT_DIR/bin -name "$SNAP_NAME*"|wc -l)"
+    SNAP_MOUNT_DIR="$(os.paths snap-mount-dir)"
+    N="$(find \"$SNAP_MOUNT_DIR/bin\" -name "$SNAP_NAME*"|wc -l)"
```

I think this should be:
```suggestion
    N="$(find "$SNAP_MOUNT_DIR/bin" -name "$SNAP_NAME*"|wc -l)"
```